simplified-app-openebooks/src/main/res/values/features.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources>
+    <!-- The configurable application name -->
+    <string name="feature_app_name">Open eBooks</string>
+    <!-- The configurable application icon -->
+    <drawable name="feature_app_icon">@drawable/oei</drawable>
+    <!-- The configurable splash screen image -->
+    <drawable name="feature_app_splash">@drawable/oei_splash</drawable>
+    <!-- The configurable main color, used for branding. -->
+    <color name="feature_main_color">#8cc63e</color>
+    <!-- The configurable secondary color, used for branding. -->
+    <color name="feature_secondary_color">#7baf38</color>
+    <!-- The default Adobe vendor ID -->
+    <string name="feature_adobe_vendor_id">OEI</string>
+</resources>
```

Comment:
I'm not completely sure if any of these `feature_` settings are still used. These were pre-LFA, even...

Reply:
Hah, yeah, I had no idea what these were for. Happy to delete them if they're unused.